src/sqs/discovery.ts

```diff
@@ -1,4 +1,12 @@
+import {MessageHandler} from "./index"
+import app from "../worker/app";
+import {discovery} from "../sync/discovery";
+
 export type DiscoveryMessagePayload = {
 	installationId: number,
 	jiraHost: string
 }
+
+export const discoveryQueueMessageHandler : MessageHandler<DiscoveryMessagePayload> = async (context) => {
+	await discovery(app)({data: context.payload}, context.log);
```

Comment:
The only thing this queue is doing is calling the existing handler for Discovery queue.

We are not using any of its middleware, because:
1) We don't use Sentry anyway
2) We handle timeouts in our error handler